docs/dev-setup/install.md

```diff
@@ -107,13 +107,30 @@ go test github.com/hyperledger/fabric/core/container -run=BuildImage_Peer
 go test github.com/hyperledger/fabric/core/container -run=BuildImage_Obcca
 ```
 
-## Building outside of Vagrant <a name="vagrant"></a>
-While not recommended, it is possible to build the project outside of Vagrant (e.g., for using an editor with built-in Go toolking). In such cases:
+## Building outside of Vagrant
+It is possible to build the project outside of Vagrant (e.g. when one already runs Linux and wants to develop natively). Generally speaking, one has to match the vagrant [setup file](https://github.com/hyperledger/fabric/blob/master/devenv/setup.sh).
```

Comment:
This text seems to imply that it will only run natively on Linux. 
"when one already runs Linux and wants to develop natively"
That is not true. Maybe just "when one wants to develop or run natively"